ERCS/erc-7893.md

```diff
@@ -495,7 +525,7 @@ Based on conducted tests, it is recommended:
    - Batch updates for multiple tokens
    - Limit array sizes in updates
 
-For more details please visit: [Solvency Proof Implementation](../assets/eip-7893/SolvencyProof.sol)
+A reference implementation is available in the author's GitHub repository.
```

Comment:
```suggestion
```

We don't allow external links. If you'd like to bring back a minimal version of your reference implementation, you may.